controllers/openstackmachine_controller.go

```diff
@@ -224,28 +224,27 @@ func (r *OpenStackMachineReconciler) reconcileDelete(ctx context.Context, logger
 
 	if instanceStatus == nil {
 		logger.Info("Skipped deleting machine that is already deleted")
```

Comment:
maybe add instance ID here?

Reply:
I added this, but then removed it again because after looking at how we create the log object I realised it's redundant. We don't have instance ID here because the machine has been deleted. I had added machine name, but that's already included by the logger: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/cf5e5cb826db47cf79ccd3d8bdd2a779f3c584c0/controllers/openstackmachine_controller.go#L88